src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -54,51 +54,104 @@
-						$event ? 1 : 0
-					)"
-				>
-					I have read and agree to the
-					<a
-						:href="`https://${this.$appConfig.host}/legal/terms`"
-						target="_blank"
-						title="Open Terms of Use in a new window"
-					>Terms of Use</a>
-					and
-					<a
-						:href="`https://${this.$appConfig.host}/legal/privacy`"
-						target="_blank"
-						:title="`Open Privacy ${enableCommsExperiment ? 'Notice' : 'Policy' } in a new window`"
-					>Privacy {{ enableCommsExperiment ? 'Notice' : 'Policy' }}</a>.
-					<p v-if="$v.termsAgreement.$error" class="input-error tw-text-danger tw-text-base">
-						You must agree to the Kiva Terms of service & Privacy
-						{{ enableCommsExperiment ? 'Notice' : 'Policy' }}.
+				<template v-if="enableRadioBtnExperiment">
+					<fieldset class="tw-flex tw-flex-col tw-gap-2 tw-mt-1 tw-mb-2 tw-text-small">
+						<kv-radio
+							value="1"
+							name="reportComms"
+							v-model="selectedComms"
+							v-kv-track-event="[
+								'basket',
+								'click',
+								'marketing-updates',
+								// eslint-disable-next-line max-len
+								'Receive email updates from Kiva (including borrower updates and promos). You can unsubscribe anytime.',
+							]"
+							:class="{'radio-error': $v.selectedComms.$error}"
+						>
+							<!-- eslint-disable-next-line max-len -->
+							Receive email updates from Kiva (including borrower updates and promos). You can unsubscribe anytime.
+						</kv-radio>
+						<kv-radio
+							value="0"
+							name="reportComms"
+							v-model="selectedComms"
+							:class="{'radio-error': $v.selectedComms.$error}"
+							v-kv-track-event="[
+								'basket',
+								'click',
+								'marketing-updates',
+								// eslint-disable-next-line max-len
+								'Dont receive email updates from Kiva (including borrower updates and promos). You can unsubscribe anytime.',
+							]"
+						>
+							<!-- eslint-disable-next-line max-len -->
+							Don't receive email updates from Kiva (including borrower updates and promos). You can unsubscribe anytime.
+						</kv-radio>
+					</fieldset>
+					<p
+						v-if="$v.selectedComms.$error"
+						class="input-error tw-text-danger tw-text-base tw-mb-2 tw-text-small"
+					>
+						Choose your communication preferences.
 					</p>
-				</kv-checkbox>
-				<kv-checkbox
-					data-testid="basket-guest-email-updates"
-					id="emailUpdates"
-					class="checkbox tw-text-small tw-mb-2"
-					name="emailUpdates"
-					v-model="emailUpdates"
-					@update:modelValue="$kvTrackEvent(
-						'basket',
-						'click-marketing-updates',
-						emailUpdatesCopy,
-						$event ? 1 : 0
-					)"
-				>
-					{{ emailUpdatesCopy }}
-				</kv-checkbox>
+					<p
+						v-if="selectedComms === '0'"
+						class="tw-border-brand-200 tw-border tw-bg-brand-100 tw-p-1.5 tw-rounded tw-text-small"
+					>
+						Can we ask you to reconsider? This borrower and others like them will need your
+						help to change their lives. You can unsubscribe at any time.
```

Comment:
Does design want to adjust the spacing or copy so that "time" doesn't sit on a new line by itself?